src/Codeception/Lib/Parser.php

```diff
@@ -25,13 +30,13 @@ public function __construct(Scenario $scenario, Metadata $metadata)
         $this->metadata = $metadata;
     }
 
-    public function prepareToRun($code)
+    public function prepareToRun($code): void
     {
         $this->parseFeature($code);
         $this->parseScenarioOptions($code);
     }
 
-    public function parseFeature($code)
+    public function parseFeature($code): void
```

Comment:
It seems that you forgot to add types to method parameters in this file.
I see lots of strings here.